codegen/src/main/java/software/amazon/awssdk/codegen/AddMetadata.java

```diff
@@ -50,18 +47,17 @@ public static Metadata constructMetadata(ServiceModel serviceModel,
         String serviceName;
         String rootPackageName;
 
-        // API Gateway uses additional codegen.config settings
         if (serviceMetadata.getProtocol().equals(Protocol.API_GATEWAY.getValue())) {
-            // TODO: The meaning of root package name has changed a bit since this code was written. Specifically, the root for
-            // AWS no longer includes the service name. This changed the behavior of the API gateway generation, but we're not
-            // keeping it up to date at this time. Just be aware this has happened when updating the API gateway code.
-            serviceName = codeGenConfig.getInterfaceName();
-            rootPackageName = codeGenConfig.getPackageName();
-
-            metadata.withDefaultEndpoint(codeGenConfig.getEndpoint())
-                    .withDefaultEndpointWithoutHttpProtocol(
-                            Utils.getDefaultEndpointWithoutHttpProtocol(codeGenConfig.getEndpoint()))
-                    .withDefaultRegion(codeGenConfig.getDefaultRegion());
+            // TODO: We don't use codegen config for aws services in v2. Revisit this code when we support
+            // API gateway generation in V2
```

Comment:
Should we just remove the commented-out lines? Doesn't seem to be beneficial to keep them here. We can use git history to see the old code if we want.

Reply:
Done